test/gurka/test_time_tracking.cc

```diff
@@ -24,6 +24,9 @@ TEST(TimeTracking, make) {
   // need to access the tiles
   baldr::GraphReader reader(map.config.get_child("mjolnir"));
 
+  // this is what the default should be, with constrained second of day
+  baldr::TimeInfo basic_ti{false, 0, 0, baldr::kConstrainedFlowSecondOfDay};
```

Comment:
now timeinfo defaults to constrained time of day even when the route is not time dependent